frontend/ui/widget/menu.lua

```diff
@@ -698,6 +698,7 @@ function Menu:init()
         callback = function() self:onReturn() end,
         bordersize = 0,
         show_parent = self,
+        readonly = self.return_arrow_propagation,
```

Comment:
So, this return button will not work anymore? I don't see it being re-enabled.
I see it being used in the Statistics navigation. Anywhere else?

Reply:
Yes, It's also used in OPDS browser.